### PR TITLE
Prometheus Scraping of new Auth Service metrics

### PR DESCRIPTION
**Pull request by dilallkx**
**CHANGES:**
- General:
  - configurable grafana dashboards
  - meepctl version bump to v1.6.5
- Grafana:
  - dashboard version & path updates
  - Configurable dashboard list
-  Auth Service:
  - New /metrics endpoint to enable Prometheus scraping
  - New custom metrics
  - Prometheus golang client integration

**TESTING:**:
- UT, system & cypress tests pass

### CODE REVIEW

File: .meepctl-repocfg.yaml [hunk 1]
```diff
@@ -82,6 +82,13 @@ repo:
         # OAuth k8s secret (data: client-id, secret)
         secret: meep-oauth-gitlab
 
+    # Default monitoring dashboards
```
Comment (roymx) [inline .meepctl-repocfg.yaml:85]: I was under the impression that some of the dashboards would be avail on AdvantEDGE
Ex. node exporter, k8s exporter, etc.

File: go-packages/meep-sessions/session-store.go [hunk 2]
```diff
@@ -209,6 +213,7 @@ func getUserEntryHandler(key string, fields map[string]string, userData interfac
 		s.Sandbox = fields[ValSandbox]
 		s.Role = fields[ValRole]
 		s.Timestamp, _ = time.Parse(time.RFC3339, fields[ValTimestamp])
+		s.StartTime, _ = time.Parse(time.RFC3339, fields[ValStartTime])
```
Comment (roymx) [inline go-packages/meep-sessions/session-store.go:216]: Do we have a metric on the Sandbox creation duration?

**Review by roymx — APPROVED** [review]

File: .meepctl-repocfg.yaml (hunk 1 above)
Comment (dilallkx) [reply .meepctl-repocfg.yaml:85]: Agreed. I moved the Node Exporter dashboard back this this repo.

File: go-packages/meep-sessions/session-store.go (hunk 2 above)
Comment (dilallkx) [reply go-packages/meep-sessions/session-store.go:216]: Currently not. This is a simple metric to add if we need it.

**Pull request merged**